addons/account/views/request_document_template_view.xml

```diff
@@ -0,0 +1,116 @@
+<tbody>
+    <!-- HEADER -->
+    <tr>
+        <td align="center" style="min-width: 590px;">
+            <table border="0" cellpadding="0" cellspacing="0" width="590" style="min-width: 590px; background-color: white; padding: 0px 8px 0px 8px; border-collapse:separate;">
+                <tr><td valign="middle">
+                    <span style="font-size: 10px;">
+                        Document Request: <br/>
+                        <t t-if="requested_document.name">
+                            <span style="font-size: 20px; font-weight: bold;" t-out="requested_document.name or ''" />
+                        </t>
+                    </span><br/>
+                </td><td valign="middle" align="right" t-if="not requested_document.create_uid.company_id.uses_default_logo">
+                    <img t-attf-src="/logo.png?company={{ requested_document.create_uid.company_id.id }}" style="padding: 0px; margin: 0px; height: auto; width: 80px;" t-att-alt="requested_document.create_uid.company_id.name"/>
+                </td></tr>
+                <tr><td colspan="2" style="text-align:center;">
+                    <hr width="100%" style="background-color:rgb(204,204,204);border:medium none;clear:both;display:block;font-size:0px;min-height:1px;line-height:0; margin: 16px 0px 16px 0px;"/>
+                </td></tr>
+            </table>
+        </td>
+    </tr>
+    <!-- CONTENT -->
+    <tr>
+        <td align="center" style="min-width: 590px;">
+            <table border="0" cellpadding="0" cellspacing="0" width="590" style="min-width: 590px; background-color: white; padding: 0px 8px 0px 8px; border-collapse:separate;">
+                <tr><td valign="top" style="font-size: 13px;">
+                    <div>
+                        <t t-out="request_message" />
+                        <br/><br/>
+                        <center>
+                            <div style="margin: 16px 0px 16px 0px;">
+                                <a t-att-href="requested_document.full_url"
+                                    style="background-color: #875A7B; padding: 20px 30px 20px 30px; text-decoration: none; color: #fff; border-radius: 5px; font-size:13px;">
+                                    Upload the requested document
+                                </a>
+                            </div>
+                        </center><br/>
+                        Please provide us with the missing document.
+                        <t t-if="user and user.signature">
+                            <br/>
+                            <t t-out="user.signature or ''" />
+                            <br/>
+                        </t>
+                    </div>
+                </td></tr>
+                <tr><td style="text-align:center;">
+                    <hr width="100%" style="background-color:rgb(204,204,204);border:medium none;clear:both;display:block;font-size:0px;min-height:1px;line-height:0; margin: 16px 0px 16px 0px;"/>
```

Comment:
I think if this line is split into several lines, it will just increase the number of lines and make the code harder to read without any benefit as it's just a horizontal break :sweat_smile: